README.md

```diff
@@ -1,16 +1,38 @@
-# Title
+# Build a Simple Calculator
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Use Ruby math operations to build a calculator
 
 ## Introduction
 
-## SWBAT 1
+Calculators can be very useful devices in day-to-day activities. You've likely
+used a calculator to  add up bills for this month or calculate tip at a
+restaurant. We're going to take our arithmetic knowledge and put it to the test
+by writing functions that will do basic math calculations for us, just like we
+can see in IRB.
 
-## SWBAT 2
+## Use Ruby Math Operations to Build a Calculator
+
+Fork and clone this repo and open `lib/math.rb`. You'll find a bunch of empty
+methods that take numbers as arguments. Build the appropriate behavior for each
+of the following methods:
+- `addition` - Build the method `addition` that adds `num1` to `num2`
```

Comment:
Be careful here, do we expect them to evaluate the expression (i.e. return the value of the expression) or return arg1 (operation) arg2:

```ruby
arg1 - arg2 #=> 2

# or

arg1 -= arg2
```

Learners are still pretty new and might get confused by this subtlety. Let's help them out by stating our expectation explicitly.